go.mod

```diff
@@ -202,6 +201,7 @@ require (
 
 // SourceNetwork fork og graphql-go
 replace (
+	github.com/dgraph-io/badger/v4 => github.com/fredcarle/badger/v4 v4.0.0-20230801145501-d3a57bd4c2ec
```

Comment:
question: I thought we agreed to use a source fork instead of a personal one? I do have a preference for a sourcenetwork fork.

Reply:
Used mine because I didn't have the rights to push to our fork. I can update it now that I do have rights but it's temporary anyways.

Reply:
Cheers :)